### ISSUE TITLE
CSS Stylesheet validation not showing warnings (but Jigsaw shows them)

### ISSUE THREAD

**Issue by jaimeiniesta** [validator/validator#971]
Hello! I would like to know the differences between the CSS validator at http://jigsaw.w3.org/css-validator/ and the one included with this validator. I think they're basically the same, but I'm getting different results.

I've checked [this stylesheet](https://d2bcamzsryg1pm.cloudfront.net/css/app-95ce18077c325a1d7252edb7c4707c89.css?vsn=d), and these are the [results on Jigsaw](http://jigsaw.w3.org/css-validator/validator?uri=https%3A%2F%2Fd2bcamzsryg1pm.cloudfront.net%2Fcss%2Fapp-95ce18077c325a1d7252edb7c4707c89.css%3Fvsn%3Dd&profile=css3svg&usermedium=all&warning=yes&vextwarning=&lang=en) and the [results in validator 20.3.16](http://159.65.102.161:8888/?doc=https%3A%2F%2Fd2bcamzsryg1pm.cloudfront.net%2Fcss%2Fapp-95ce18077c325a1d7252edb7c4707c89.css%3Fvsn%3Dd).

Basically, they return the same 3 errors, but Jigsaw also returns 918 warnings, while validator returns no warnings.

I've seen that in Jigsaw there is an option to show warnings (yes/no), but this doesn't seem to work in validator, it always excludes warnings.

Why is that? Is the CSS checker included in validator the most current? Is there some option I'm missing?

Thanks!

**Comment by sideshowbarker** [comment]
> Is the CSS checker included in validator the most current?

Yes

> I've seen that in Jigsaw there is an option to show warnings (yes/no), but this doesn't seem to work in validator, it always excludes warnings.

Yes, that’s by design. The HTML checker drops all CSS warnings on the floor.

> Is there some option I'm missing?

You’re not missing anything. There’s intentionally no option in the HTML checker to show CSS warnings, and I have no plans to add such an option.

Part of the design philosophy of the Nu checker has been to keep warnings to an absolute minimum. But the CSS backend was designed with a different philosophy toward warnings — and generates way, way more than what anybody would reasonably consider the absolute minimum.

The full set of CSS warnings for a particular resource can be seen by using the CSS validator rather than the Nu checker.

**Comment by jaimeiniesta** [comment]
Thanks a lot for the explanation! I think that's a wise decision to make, the amount of warnings output was overwhelming.
